vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewFragment.kt

```diff
@@ -96,12 +98,29 @@ class SessionOverviewFragment :
                 ?.setTitle(titleResId)
     }
 
-    private fun updateEntryDetails(deviceId: String) {
+    private fun updateSessionInfo(deviceId: String) {
         views.sessionOverviewEntryDetails.setOnClickListener {
             viewNavigator.navigateToSessionDetails(requireContext(), deviceId)
         }
     }
 
+    private fun updatePushNotificationToggle(deviceId: String, pushers: List<Pusher>) {
+        views.sessionOverviewPushNotifications.apply {
+            if (pushers.isEmpty()) {
+                isVisible = false
+            } else {
+                val allPushersAreEnabled = pushers.all { it.enabled }
+                setOnCheckedChangeListener { _, _ -> }
```

Comment:
I understand why this is needed. I think setting listener as null would seem cleaner `setOnCheckedChangeListener(null)`.